src/qml/RestartServicesDialog.qml

```diff
@@ -39,21 +43,44 @@ Dialog {
     id: container
     onAccepted: PatchManager.restartServices()
 
-    SilicaFlickable {
-        anchors.fill: parent
-        Column {
-            spacing: Theme.paddingMedium
-            anchors.left: parent.left; anchors.right: parent.right
-            DialogHeader {
-                acceptText: qsTranslate("", "Restart services")
-            }
+    Component.onCompleted: console.info("Will restart " + PatchManager.appsToRestart);
 
-            Label {
-                anchors.left: parent.left; anchors.leftMargin: Theme.paddingMedium
-                anchors.right: parent.right; anchors.rightMargin: Theme.paddingMedium
-                wrapMode: Text.WordWrap
-                color: Theme.highlightColor
-                text: qsTranslate("", "Some services will be restarted now. Reloading the homescreen of the device might take a little time.")
+    Column {
+        spacing: Theme.paddingSmall
+        width: parent.width
+        DialogHeader {
+            acceptText: qsTranslate("", "Restart")
+        }
+        Label {
+            anchors.horizontalCenter: parent.horizontalCenter
+            width: parent.width -  Theme.horizontalPageMargin * 2
+            color: Theme.highlightColor
+            text: qsTranslate("", "Some services will be restarted now. Reloading the homescreen of the device might take a little time.")
+            wrapMode: Text.WordWrap
+            horizontalAlignment: Text.AlignJustify
+        }
+        SectionHeader { text: qsTranslate("", "List of services:" ); color: Theme.secondaryHighlightColor }
+        Column {
+            id: col
+            anchors.horizontalCenter: parent.horizontalCenter
+            width: parent.width -  Theme.horizontalPageMargin * 2
+            Repeater {
+                model: PatchManager.appsToRestart
+                delegate: Component { TextSwitch {
+                    text: qsTranslate("Sections", modelData)
```

Comment:
This works for all languages but English, because English doesn't have a dedicated translation and so uses the (lowercase) original strings.

Reply:
Nah, totally minor issue that just came up along the way.

Reply:
Thanks for fixing this!

Reply:
Just to document that Transifex uses "translated" strings from the source file as source strings, when the are available (which makes sense IMO).
Thus this change changed the source strings of all 14 PM-categories for all 14 translations.  And because Transifex does not allow programmatic changes at Transifex (e.g., per RegEx), I manually adapted these 14 terms for 13 translations (all, but zh).  Maybe scripted downloading, programmatic changing the strings locally and scripted uploading would have been easier, but that would have required learning how to use Transifex's web API, which I wanted to avid this time.

Reply:
@nephros, please use the exclamation mark only if something went wrong.
Unfortunately the classic conventions for writing dialogues are less obeyed than in the early 2000s, but I still would like to stick to them.
* **!** means: Attention, something went wrong.
* **...** (concluding, i.e., at the end of a phrase or sentence) means: A new window is going to be opened.   This is most suitable for choices (which would open a new window).
* There are more, I do not remember spontaneously.

The dialogues in MS-Windows and most of KDE and Gnome are adhering to these conventions, so people are used to them (even most not consciously).